browser/brave_screenshots/screenshots_tab_feature.cc

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/brave_screenshots/screenshots_tab_feature.h"
```

Comment:
filename and classname should match and also naming should be consistent so this should be `brave_screenshots_tab_feature`